src/tests/cwrapper/test_cwrapper.c

```diff
@@ -41,7 +41,19 @@ void test_cwrapper() {
     printf("Rational : %s\n", s);
     printf("Is_a_Symbol %s: %d\n", s, is_a_Symbol(e));
     printf("Is_a_Rational %s: %d\n", s, is_a_Rational(e));
-    printf("Is_a_Integer %s: %d\n", s, is_a_Integer(e));
+    printf("**********Is_a_Integer %s: %d\n", s, is_a_Integer(e));
+
+	integer_set_ui(e, 123);
```

Comment:
Please fix the indentation.
Also you can remove the excess '*' from the `printf`